File: language_compare/loop1.py
import time

start = time.time()
total = 0

for x in range(1000000000):
    total += x
else:
    print ("Count reached: " + str(total))

end = time.time()

#Subtract Start Time from The End Time
total_time = end - start
print("Time taken by program in Python is: "+ str(total_time), "seconds")

# rig: Dell PowerEdge T110-II (3.1 GHz/core x4 - 4GB Ram)
# Time taken by program in Python is: 96.16518974304199 seconds

File: language_compare/Loop1.java
class Loop1 {
    public static void main(String args[]) {
        long startTime = System.nanoTime();
        long j = 0;

        for (long i = 0; i < 1000000000; i++) {
            j += i;
            //System.out.println(i);
        }

        System.out.println("Count reached: " + j);

        long elapsedTime = System.nanoTime() - startTime;
        System.out.println("Time taken by program in JAVA in ms is: " + elapsedTime / 1000000);
    }
}

// rig: Dell PowerEdge T110-II (3.1 GHz/core x4 - 4GB Ram)
// Time taken by program in JAVA in ms is: 600

